docs/core-concepts/components.md

```diff
@@ -51,7 +51,7 @@ The karmada-controller-manager runs various custom controller processes.
 
 The controllers watch Karmada objects and then talk to the underlying clusters' API servers to create regular Kubernetes resources.
 
-The controllers are listed at [Karmada Controllers](../administrator/configuration/configure-controllers/#karmada-controllers).
```

Comment:
This link works on my side: https://karmada.io/docs/administrator/configuration/configure-controllers/#karmada-controllers.

@XiShanYongYe-Chang How about you?

Reply:
Hey, website seems to be working for me too now (maybe I was reading an older docs version?). Link only breaks down when you try to access the docs via github (from here for example: https://github.com/karmada-io/website/blob/main/docs/core-concepts/components.md). Would you suggest that to be fixed or it's ok as long as the website is functional?

Reply:
The link format is used everywhere on the website, I don't think it's worth it to fix them all.
But thanks for spotting, it's my first time trying to navigate the doc from the markdown file though. :)

Reply:
Agreed! Since website is ok, I also think there's no reason to fix them. Glad to help! I will proceed to close this PR

Reply:
Thank you all the same. Looking forward to your contribution.